### ISSUE TITLE
dataset googledrive link is 404

### ISSUE THREAD

**Issue by Molers** [Aatr0x13/MC-Denoising-via-Auxiliary-Feature-Guided-Self-Attention#1]

**Comment by Chuudy** [comment]
@Molers Did you manage to acquire the proper training data from somewhere else?

**Comment by Molers** [comment]
> @Molers Did you manage to acquire the proper training data from somewhere else?

![image](https://user-images.githubusercontent.com/23693269/210934281-e69f7322-3553-4683-860c-fd42b6b016d9.png)
I think you can use this dataset to test, this paper's dataset is 404，and the author hasn't spoken in a long time
ACFM github link:
https://github.com/mcdenoising/AdvMCDenoise
![image](https://user-images.githubusercontent.com/23693269/210935254-0de586de-cc46-474d-be34-fcc9dab77835.png)

**Comment by Chuudy** [comment]
Thanks for the message. That's exactly what I am trying to do. The only problem is that the training script assumes data decomposed to specular and diffuse layers, and the data provided by ACFM does not have such data. Also, number of shots in the dataset does not agree with the above description, so I wonder whether the current dataset is even the correct one.

**Comment by Molers** [comment]
> Thanks for the message. That's exactly what I am trying to do. The only problem is that the training script assumes data decomposed to specular and diffuse layers, and the data provided by ACFM does not have such data. Also, number of shots in the dataset does not agree with the above description, so I wonder whether the current dataset is even the correct one.

I didn't use the specular and diffuse in my project,So I may not be able to help you.
![image](https://user-images.githubusercontent.com/23693269/211132162-395c3164-5151-43ab-b575-4decf5a0d83d.png)
I have modified the code, using only the Normal, the depth, the albedo, noisy. Gt,
But if you want to use the data set that contains specular and diffuse, I suggest you take a look at the paper,
![image](https://user-images.githubusercontent.com/23693269/211132251-209193b4-7365-4706-b46c-0096b5bc1126.png)
Use kalantari or bako or Xu's paper dataset.Otherwise you have to download the scene and generate some data sets yourself.

**Comment by Molers** [comment]
I also used ACFM datasets and ran the code with good results

**Comment by Chuudy** [comment]
Hey @Molers, thanks for your tips. I came up with similar conclusions yesterday, and it seems it's training so far.
Hopefully, other people in the future will have easier time running this thing.

**Comment by Git-XLAN** [comment]
> Hey @Molers, thanks for your tips. I came up with similar conclusions yesterday, and it seems it's training so far. Hopefully, other people in the future will have easier time running this thing.
No!the future you hope for is not comming!I am getting struck in dataset problems.How should I use only Normal,Depth,Albedo,Noisy and gt to train the network?could you help me please!
